spec/bounds1.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
Think a more descriptive name than `bounds1` here would be useful before we end up with a list of 
```
bounds1
bounds2
bounds3
...
```